CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
  - Fixing BaseGame tap detectors issues
  - Upgrade Flutter SVG to fix for flame web
  - Add linting to all the examples
+ - Run linting only on affected and changed examples
```

Comment:
I don't know if we should add those "internal" kind of stuff on changelog, what do you guys think?

Reply:
I was thinking that too, doesn't really add any value. But I saw that it had been done before so I added mine too. Should I remove it?

Reply:
Yeah, it has been done before, but I always questioned in my head if we should continue doing it. Lets hear what @luanpotter thinks

Reply:
I don't think we need to enforce it but I don't see why we can't have it (since it's already there).

The reason I added mine is because it was a required item on the PR checklist

Reply:
Maybe we could change the checklist to have an if :P

Reply:
We could, sincerely I don't have a strong opinion either way. I was just following :p

Reply:
You can just remove things from the checklist per PR if you don't think they apply, like I did on this one. ;)